DSCResources/xPowerPlan/xPowerPlan.schema.mof

```diff
@@ -0,0 +1,6 @@
+[ClassVersion("1.0.0.0"), FriendlyName("xPowerPlan")]
+class xPowerPlan : OMI_BaseResource
+{
+    [Key, Description("Use value 'Present' if the plan name should be activated. Use value 'Absent' to active the default plan for the operating system (hard-coded to plan 'Balanced')."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
```

Comment:
The powerplan cannot be both present and absent.  Please use the Single Instance best practice:

https://msdn.microsoft.com/en-us/powershell/dsc/singleinstance